package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@adobe/content-lake-extractors-shared",
   "version": "1.9.0",
-  "description": "Content Lake - Shared",
+  "description": "Content Lake - Extractors Shared",
```

Comment:
maybe remove `Rename package from @adobe/content-lake-extractors-shared to @adobe/content-lake-shared` from the description :)